samples/bluetooth/central_hr/prj_minimal.conf

```diff
@@ -0,0 +1,103 @@
+CONFIG_BT=y
+CONFIG_BT_SMP=y
+CONFIG_BT_CENTRAL=y
+
+# Drivers and peripherals
+CONFIG_I2C=n
+CONFIG_WATCHDOG=n
+CONFIG_PINCTRL=n
+CONFIG_SPI=n
+CONFIG_GPIO=n
+CONFIG_SERIAL=n
+
+# Power management
+CONFIG_PM=n
+
+# Interrupts
+CONFIG_DYNAMIC_INTERRUPTS=n
+CONFIG_IRQ_OFFLOAD=n
+
+# Memory protection
+CONFIG_THREAD_STACK_INFO=n
+CONFIG_THREAD_CUSTOM_DATA=n
+CONFIG_FPU=n
+
+# Boot
+CONFIG_BOOT_BANNER=n
+CONFIG_BOOT_DELAY=0
+
+# Console
+CONFIG_CONSOLE=n
+CONFIG_UART_CONSOLE=n
+CONFIG_STDOUT_CONSOLE=n
+CONFIG_PRINTK=n
+CONFIG_EARLY_CONSOLE=n
+
+# Build
+CONFIG_SIZE_OPTIMIZATIONS=y
```

Comment:
Zephyr Controller library overrides it to use speed optimization of Controller source code. But lets keep it like that until a day we absolutely need to fit the overgrowth!

Reply:
everywhere?
I know SDC's build system forces speed only for time-critical files like stuff that runs in Tifs.